### ISSUE TITLE
Notebook folding misplaces cell selection

### ISSUE THREAD

**Issue by kieferrm** [microsoft/vscode#107763]
Issue Type: <b>Bug</b>

![5IH6f4ggFg](https://user-images.githubusercontent.com/4674940/94629797-9c819080-0278-11eb-830d-2eb7ebd0edef.gif)

1) Have the keyboard focus on a cell that is a "child" of a foldable markdown cell
2)<kbd>ArrowLeft</kbd>
-> I agree that this should fold the section, but now the "parent" should have the keyboard focus. However, it's lost and on <kbd>ArrowDown</kbd> keyboard focus goes to the first cell of the notebook.

VS Code version: Code - Insiders 1.50.0-insider (0580328b728a69768558cc8242067cae7b6a1c21, 2020-09-29T09:05:00.729Z)
OS version: Darwin x64 18.7.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-4870HQ CPU @ 2.50GHz (8 x 2500)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|2, 1, 1|
|Memory (System)|16.00GB (0.30GB free)|
|Process Argv|-psn_0_10267082 --crash-reporter-id d4b72415-0109-45b3-a1a7-967fdd12f0f4 --crash-reporter-id d4b72415-0109-45b3-a1a7-967fdd12f0f4 --crash-reporter-id d4b72415-0109-45b3-a1a7-967fdd12f0f4|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (37)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-track-build-errors|aes|0.0.2
git-branch|ale|1.6.0
github-markdown-preview|bie|0.0.2
markdown-checkbox|bie|0.1.3
markdown-emoji|bie|0.0.9
markdown-preview-github-styles|bie|0.1.6
markdown-yaml-preamble|bie|0.0.4
vscode-svgviewer|css|2.0.0
vscode-eslint|dba|2.1.9
gitlens|eam|10.2.2
tsl-problem-matcher|eam|0.3.0
vsc-material-theme|Equ|33.0.0
vsc-material-theme-icons|equ|1.2.0
graphviz-markdown-preview|gee|0.0.8
vscode-pull-request-github-insiders|Git|2020.9.41222
mdmath|goe|2.5.1
crazy|kie|0.0.2
svgpreview|kis|0.2.0
rainbow-csv|mec|1.7.1
theme-monokai-pro-vscode|mon|1.1.17
vscode-docker|ms-|1.6.0
python|ms-|2020.9.112786
remote-containers|ms-|0.143.0
remote-ssh-edit-nightly|ms-|2020.9.41220
remote-ssh-nightly|ms-|2020.9.41220
remote-wsl|ms-|0.50.0
azure-account|ms-|0.9.3
azurecli|ms-|0.5.0
js-debug-nightly|ms-|2020.9.2817
vscode-github-issue-notebooks|ms-|0.0.52
vscode-js-profile-flame|ms-|0.0.10
vscode-markdown-notebook|ms-|0.0.10
vsonline|ms-|1.0.2823
material-icon-theme|PKi|4.3.0
ruby|reb|0.27.0
vscode-xml|red|0.13.0
vscode-ruby|win|0.27.0

(4 theme extensions excluded)

</details>

**Comment by rebornix** [comment]
We should focus the markdown cell which contains the folded cell.
